.github/workflows/run_tests_suite_python39.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
-          pip install tensorflow==2.11.*
-          pip install torch
+          pip install tensorflow==2.12.*
+          pip install torch==1.13.*
```

Comment:
Maybe it's better to write a single yml which gets the python version so 3.9 and 3.10 will use it instead of duplicating them?